framework/wazuh/core/cluster/worker.py

```diff
@@ -233,15 +233,15 @@ def process_request(self, command: bytes, data: bytes) -> Union[bytes, Tuple[byt
                 asyncio.create_task(
                     self.server.local_server.clients[dapi_client.decode()].send_request(command, error_msg))
             except WazuhClusterError:
-                raise WazuhClusterError(3025)
+                raise WazuhClusterError(3018)
```

Comment:
Error 3025 has been removed and we have changed to error 3018, whose message is "Error sending request", can't we be more specific in case of error in this part of the code?

Reply:
Well actually we are only capturing `WazuhClusterError` and raising it again after changing the error code. The `send_request()` function already raises WazuhClusterErrors with more specific information, so I think the best will be to remove this Try/Except.